cypress.json

```diff
@@ -1,5 +1,8 @@
 {
   "projectId": "qa7fe2",
   "ignoreTestFiles": "*.js",
-  "baseUrl": "http://localhost:3000"
+  "baseUrl": "http://localhost:3000",
+  "env": {
+    "RETRIES": 2
+  }
```

Comment:
Also, not related to `firefox` as a test runner, but unless I'm missing something, does anyone think `retries` are actually occurring? hopefully, we can see some evidence that this works